src/components/Email.js

```diff
@@ -18,11 +18,10 @@ export default class Email extends React.Component {
                 <a href={`mailto:${email}`}>
                     <icons.Email />
                     <div className="Contact-text">
-                        <span className="kind">
-                            {this.props.comment ? this.props.comment
-                            : "Email"}
-                        </span>
-                        {' '}
+				        <span className="kind">
+				            {this.props.comment ? this.props.comment
+				            : ""}
```

Comment:
I think you need something here still, even if it's marked as Screenreader only.